src/components/Alteration/AlterationSummary.vue

```diff
@@ -192,6 +192,7 @@ export default class AlterationSummary extends Mixins(
   @Getter isBusySaving!: boolean
   @Getter getFlagsReviewCertify!: FlagsReviewCertifyIF
   @Getter haveNewResolutionDates!: boolean
+  @Getter areProvisionsRemoved!: boolean
```

Comment:
This getter is used in this component but not declared. In this situation, usually the getter is imported from a mixin, but I prefer to have it declared locally.